tools/bwa.py

```diff
@@ -210,9 +210,8 @@ def mem(self, inReads, refDb, outAlign, options=None, min_score_to_filter=None,
         options = [] if not options else options
 
         if '-t' not in options:
-            options.extend(('-t', str(threads)))
+            options.extend(('-t', str(util.misc.sanitize_thread_count(threads))))
 
-        threads = threads or util.misc.available_cpu_count()
```

Comment:
Actually, watch out about removing this line entirely, down below you're actually using the `threads` variable again for `samtools.sort`. Perhaps just precede the `if` block with `threads = util.misc.sanitize_thread_count(threads)` and then you can use that in both places.